src/main/java/it/pagopa/pagopa/apiconfig/service/RefreshService.java

```diff
@@ -0,0 +1,64 @@
+package it.pagopa.pagopa.apiconfig.service;
+
+import it.pagopa.pagopa.apiconfig.exception.AppError;
+import it.pagopa.pagopa.apiconfig.exception.AppException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.client.HttpClientErrorException;
+import org.springframework.web.client.RestTemplate;
+import static org.springframework.http.HttpMethod.*;
+
+@Service
+@Validated
+public class RefreshService {
+    public enum ConfigDomain {
+        FTP_SERVER, INFORMATIVA_CDI, INFORMATIVA_PA, PA, PDD, PSP, Global
+    }
+
+    public enum JobTrigger {
+        paInviaRt, paRetryPaInviaRtNegative, paInviaRtRecovery, paSendRt
+    }
+
+    @Autowired
+    private RestTemplate restTemplate;
+
+    @Value("${service.trigger.job.host}")
+    private String jobTriggerUrl;
+
+    @Value("${service.refresh-config.domains.host}")
+    private String refreshConfigUrl;
+
+    public String jobTrigger(JobTrigger jobType) {
+        return get(jobTriggerUrl + "/" + jobType).getBody();
+    }
+
+    public String refreshConfig(ConfigDomain configType) {
+        String url;
+        if(configType.equals(ConfigDomain.Global))
+          url = jobTriggerUrl + "/refreshConfiguration";
+        else url = refreshConfigUrl + "/" + configType;
+
+        return get(url).getBody();
+    }
+
+    private ResponseEntity<String> get(String url) {
```

Comment:
I don't like this solution. Is it possible to use [FeignClient](https://github.com/pagopa/pagopa-afm-utils/blob/031435fd4b9a157572fa040cb90bed6580456e9e/src/main/java/it/gov/pagopa/afm/utils/service/MarketPlaceClient.java#L17)? If it's not possibile, I'd like to put this method in an utility class

Reply:
Yes, thanks for the hint